google-oauth-client/src/test/java/com/google/api/client/auth/openidconnect/IdTokenVerifierTest.java

```diff
@@ -39,6 +56,25 @@
   private static final String ISSUER2 = ISSUER + "2";
   private static final String ISSUER3 = ISSUER + "3";
 
+  private static final String ES256_TOKEN =
```

Comment:
Would you add source code comment about how to generate the values?

Reply:
Those are some test examples, not generated ones as I understand.